source/help/getting-started/creating-an-account.rst

```diff
@@ -0,0 +1,53 @@
+=================================================
+Creating an account and signing in the first time
```

Comment:
I'd like to reserve title case for topic or page titles, and use sentence case for sub sections.

Note also that this file was created when the style guide was in its very early phase, so I still need to make some adjustments for that.